packages/dx-scheduler-core/src/plugins/appointment-form/utils.test.ts

```diff
@@ -151,4 +152,36 @@ describe('AppointmentForm utils', () => {
         .toHaveBeenCalledWith('lastLabel');
     });
   });
+  describe('#getCountDependingOnRecurrenceType', () => {
+    it('should return count = 30 for daily appointments', () => {
+      expect(getCountDependingOnRecurrenceType(RRULE_REPEAT_TYPES.DAILY))
+        .toEqual(BASIC_DAILY_COUNT);
+    });
+    it('should return count = 13 for weekly appointments', () => {
+      expect(getCountDependingOnRecurrenceType(RRULE_REPEAT_TYPES.WEEKLY))
+        .toEqual(BASIC_WEEKLY_COUNT);
```

Comment:
We should expect `13` instead of `BASIC_WEEKLY_COUNT`. Because if constant will change, you can't know about it.